### PR TITLE
Solar system animation 

### PR DESCRIPTION
**Pull request by HossamGouda**
Solar system animation using css and Html only .

### CODE REVIEW

File: Art/HossamGouda-Rocket -Launch/index.html
Comment (LaurelineP) [inline Art/HossamGouda-Rocket -Launch/index.html:1]: ## HTML to review for css file name change
- [ ] Once you've renamed the css file according to the Readme description, please adjust the name in link tag

File: Art/HossamGouda-Rocket -Launch/style.css
Comment (LaurelineP) [inline Art/HossamGouda-Rocket -Launch/style.css:1]: ## CSS filename
- [ ] Please rename your file as mentioned in the readme.md file

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @HossamGouda and thanks for your contribution.
In order to merge your merge requests, please address the following:
- [ ] Please address the given feedbacks
- [ ] Please address the following other feedbacks
     - [ ] Your folder for the rocket launch contain a speace in the name, please remove the space

**Comment by HossamGouda** [comment]
This branch only for the Solar system animation. kindly let me know for any further requested updates !

**Comment by LaurelineP** [comment]
All good to go :) thanks for your reactivity 🙌

**Pull request merged**